internal/install/execution/install_status.go

```diff
@@ -227,6 +227,16 @@ func (s *InstallStatus) isCanceled() bool {
 	return false
 }
 
+func (s *InstallStatus) hasAnyRecipeStatus(status RecipeStatusType) bool {
```

Comment:
It looks like the other `has*()` methods could be replaced with this implementation.  Perhaps consider updating those methods as well.

Reply:
Excellent, I was hesitating converting those other usages too, I've replaced as suggested.